CONTRIBUTING.md

```diff
@@ -1,7 +1,5 @@
 # Contributing
 
-> See the [overview video](https://www.loom.com/share/609f7b61795349328730f14e1ae2166e) on how the extension works.
```

Comment:
This video is quite outdated now, thank you @matthewp for making it, it did help me back then 🫡